src/app/app.component.ts

```diff
@@ -1,10 +1,17 @@
 import { Component } from '@angular/core';
 
+export interface User {
```

Comment:
preffix conventions
I for Interfaces
E for Enum
C for constants
```suggestion
export interface IUser {
```